syndeo-template/src/main/java/com/google/cloud/syndeo/GenerateConfiguration.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.syndeo;
+
+import com.google.cloud.datapipelines.v1.BatchGetTransformDescriptionsResponse;
+import com.google.cloud.datapipelines.v1.MapType;
+import com.google.cloud.datapipelines.v1.TransformDescription;
+import com.google.cloud.datapipelines.v1.TypeName;
+import com.google.cloud.syndeo.common.ProviderUtil;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
```

Comment:
Sorry, missed this last time: Please don't star import. Only import what you intend to use.